tools/esdlcmd/esdl-publish.cpp

```diff
@@ -615,7 +599,7 @@ class EsdlDeleteESDLDefCmd : public EsdlPublishCmdCommon
                     optESDLService.set(arg);
                     break;
                 case 1:
-                    optVersionStr.set(arg);
+                    optDefinitionVersionStr.set(arg);
```

Comment:
It seems inconsistent to me that when we bind a service we use a definition ID of the format "MyDefinition.3", but here we use a separate ESDLService and Version "MyDefinition  3".

I prefer the ID type usage... which is how roxie queries work... "myquery.5".